riva/clients/nmt/streaming_s2t_client.cc

```diff
@@ -267,25 +269,30 @@ StreamingS2TClient::ReceiveResponses(std::shared_ptr<S2TClientCall> call, bool a
     gotoxy(0, 5);
   }
 
-  std::ofstream result_file(nmt_text_file_);
   while (call->streamer->Read(&call->response)) {  // Returns false when no more to read.
     call->recv_times.push_back(std::chrono::steady_clock::now());
+
+    call->latest_result_.partial_transcript = "";
+    call->latest_result_.partial_time_stamps.clear();
+
+    bool is_final = false;
     for (int r = 0; r < call->response.results_size(); ++r) {
       const auto& result = call->response.results(r);
 
+      if (result.is_final()) {
+        is_final = true;
```

Comment:
why did we add this flag? it is not used anywhere.

Reply:
This flag is required in the appendResult function